### ISSUE TITLE
No way to authenticate SIP calls

### ISSUE THREAD

**Issue by mcelicalderon** [twilio/twilio-php#324]
*Note: These issues are for bugs and feature requests for the helper libraries. If you need help or support, please email help@twilio.com and one of our experts will assist you!*

**Version:** 5.0.0-RC6
**API Subdomain (api/taskrouter/ip_messaging):** API

### Code Snippet
```php
$call = $client->account->calls->create(
    "sip:kate@example.com", "Jack",
    array(
        "url" => "http://www.example.com/sipdial.xml",
        "sipAuthPassword" => "secret",
        "sipAuthUsername" => "jack"
    )
);
```
The snippet above won't work as `sipAuthPassword` and `sipAuthUsername` will be ignored when making the request to the API. [Here](https://github.com/twilio/twilio-php/blob/next-gen/Twilio/Rest/Api/V2010/Account/CallList.php#L52-L69) is the list of the currently supported parameters.

**Comment by ihumanable** [comment]
This is an undocumented feature, we need to decide how to expose this, if we want it to be a top-level or just allow all keys through.

**Comment by jingming** [comment]
Released with `5.0.0-RC9`.
